### ISSUE TITLE
Colour schemes need updating

### ISSUE THREAD

**Issue by jeroenrnl** [jeroenrnl/zoph#67]
I have never been a big fan of the colour schemes Zoph has. I think they can use a bit of rework.

Zoph users: please leave a comment which colour scheme YOU are using.

If you use a personalized scheme you'd like to share, please do so.

**Comment by Raysir** [comment]
Hi, I am using the hoenig scheme. Not much into colors but really
    value making Zoph responsive to work with mobile devices. 
    
    Another thing that would really be nice is to have the ability to
    set the size of photos shown as individual pictures. Yes, I know how
    to click on the photo to make it larger but I am speaking of the
    default size that the individual photo first appears. With today's
    Internet speeds, I would like to have this larger without having to
    take the time to click on the photo to make it larger.
    Ray
    
    
    On 11/23/2015 01:30 PM, Jeroen Roos
      wrote:
    
    
      I have never been a big fan of the colour schemes Zoph has. I
        think they can use a bit of rework.
      Zoph users: please leave a comment which colour scheme YOU are
        using.
      If you use a personalized scheme you'd like to share, please do
        so.
      —
        Reply to this email directly or view it on
          GitHub.

**Comment by jeroenrnl** [comment]
Thanks for getting back on the colour schemes question. Please create a separate issue for each of the other requests (I can do that for you, but then you won't be registered as the creator of the issue)

**Comment by jeroenrnl** [comment]
@Raysir: The first request should be handled by issue #8, in this issue I am converting all of Zoph to templates. Once that is done, I will work on the templates to have them better work on mobile. The other idea is described in issue #11. I am aware that these issues are old and still not fixed, I have, however, limited time to work on Zoph.

**Comment by Pontusfroding** [comment]
Just because this came to my mail now I just had to test the different colour schemes. Well I can agree with you that they need some updates, I went back to default ;)  

Whish I really had time to help you with zoph because it really is the best solution out there. Hopefully I will some day.

**Comment by jeroenrnl** [comment]
@Pontusfroding Thanks for getting back, I am usually using the "beach" colour scheme. I am trying to add a few new ones in the next version, but I already found out it is quite hard to come up with a pleasing colour scheme, that is also well readable.
So far, I have come up with "steel":
![steel](https://cloud.githubusercontent.com/assets/1951972/18082095/909eb2d4-6e9d-11e6-8a7e-49075a681eb9.png)

**Comment by jeroenrnl** [comment]
@Pontusfroding Thanks for getting back, I am usually using the "beach" colour scheme. I am trying to add a few new ones in the next version, but I already found out it is quite hard to come up with a pleasing colour scheme, that is also well readable.
So far, I have come up with "steel":
![steel](https://cloud.githubusercontent.com/assets/1951972/18089148/c094654e-6ebe-11e6-9796-944b6221af31.png)
and "grey":
![grey](https://cloud.githubusercontent.com/assets/1951972/18089163/d4f37476-6ebe-11e6-9346-cfa20e3b77d2.png)

**Comment by Pontusfroding** [comment]
Thumbs up for the grey one

**Comment by jeroenrnl** [comment]
My wife came up with "happy":
![happy](https://cloud.githubusercontent.com/assets/1951972/18104723/c1a776f4-6efc-11e6-97d7-47034d8f7cf8.png)

**Comment by jeroenrnl** [comment]
Added 3 new themes, all based on the "grey" above, but with a colour accent:
![grey - accented](https://cloud.githubusercontent.com/assets/1951972/18124268/cfd84ee0-6f71-11e6-9aca-9ec9c7dda0e5.png)

**Comment by jeroenrnl** [comment]
...and modified "steel", because I thought it was a little too purple-ish:
![steel2](https://cloud.githubusercontent.com/assets/1951972/18127061/6ab07756-6f7e-11e6-83b7-cb7821d35d97.png)

**Comment by jeroenrnl** [comment]
Closing this issue now, feel free to reopen if you have comments or a custom colour scheme you'd like to have included.

@Pontusfroding thanks for your feedback!
